content/en/docs/architecture/ci-operator.md

```diff
@@ -362,6 +362,33 @@ releases:
         upper: "4.5.0-0"
 {{< / highlight >}}
 
+### Testing With a Cluster from a Cluster Pool
+Instead of using `releases` stanza in the configuration, a job can *claim* a cluster from a cluster pool and use it for testing _"something on OpenShift"_.
+
+```yaml
+- as: e2e
+  cluster_claim:
+    cloud: aws
+    product: ocp
+    owner: dpp
+    version: "4.7"
+    timeout: 1h
+  container:
+    from: cli
+  commands: |
+    printenv KUBECONFIG
+    printenv KUBEADMIN_PASSWORD_FILE
+    oc whoami
+    oc config view
+    oc get node
```

Comment:
Let's primarily use an example with `steps` (and not `container`). Most of the cluster-interacting tests are multi-stage and we want it that way. If people just blindly copy-paste contanerized tests we get increased support burden when they want to use custom secrets, parameters, dependencies etc.